dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -22,510 +30,731 @@ type Props = {
+					imagePositionOnMobile="top"
+				/>
+			</LI>
+			<LI percentage="50%">
+				<UL direction="row" wrapCards={true} showDivider={true}>
+					{remaining.map((card, cardIndex, { length }) => {
+						const columns = 2;
+						return (
+							<LI
+								key={card.url}
+								percentage="50%"
+								stretch={true}
+								showDivider={cardIndex > 0}
+								offsetBottomPaddingOnDivider={shouldPadWrappableRows(
+									cardIndex,
+									length - (length % columns),
+									columns,
+								)}
+								padSides={true}
+							>
+								<FrontCard
+									trail={card}
+									starRating={card.starRating}
+									containerPalette={containerPalette}
+									showAge={showAge}
+									imageUrl={undefined}
+									headlineSize="small"
+								/>
+							</LI>
+						);
+					})}
+				</UL>
+			</LI>
+		</UL>
+	);
+};
 
+const ColumnOfThreeCards25_ColumnOfThreeCards25_ColumnOfThreeCards25_ColumnOfThreeCards25 =
+	({
+		cards,
+		showAge,
+		containerPalette,
+	}: {
+		cards: TrailType[];
+		showAge?: boolean;
+		containerPalette?: DCRContainerPalette;
+	}) => {
+		return (
+			<UL direction="row" wrapCards={true}>
+				{cards.map((card, cardIndex, { length }) => {
+					const columns = 4;
 					return (
 						<LI
 							key={card.url}
+							percentage="25%"
+							stretch={true}
 							padSides={true}
-							percentage={
-								// Boosted primaries take up 75% of the horizontal space
-								index === boostedIndex ? '75%' : '25%'
-							}
-							showDivider={index > 0}
+							showDivider={cardIndex % columns !== 0}
+							offsetBottomPaddingOnDivider={shouldPadWrappableRows(
+								cardIndex,
+								length,
+								columns,
+							)}
 						>
 							<FrontCard
 								trail={card}
 								starRating={card.starRating}
 								containerPalette={containerPalette}
 								showAge={showAge}
-								supportingContent={card.supportingContent}
-								headlineSize={
-									index === boostedIndex ? 'large' : 'medium'
-								}
-								trailText={
-									index === boostedIndex
-										? card.trailText
-										: undefined
-								}
-								imageUrl={card.image}
-								imagePosition={
-									index === boostedIndex
-										? boostedImagePosition
-										: 'top'
-								}
-								imagePositionOnMobile={
-									index === boostedIndex ? 'top' : 'left'
-								}
-								imageSize={
-									index === boostedIndex ? 'large' : 'small'
-								}
+								imageUrl={undefined}
+								headlineSize="small"
 							/>
 						</LI>
 					);
-				}
+				})}
+			</UL>
+		);
+	};
+
+const Card25_ColumnOfCards25_ColumnOfThreeCards25_ColumnOfThreeCards25 = ({
+	cards,
+	showAge,
+	containerPalette,
+}: {
+	cards: TrailType[];
+	showAge?: boolean;
+	containerPalette?: DCRContainerPalette;
+}) => {
+	if (!cards[0]) return null;
+	const big = cards[0];
+	const remaining = cards.slice(1, 10);
+
+	return (
+		<UL direction="row" wrapCards={true}>
+			<LI
+				key={big.url}
```

Comment:
a small point, but: do we need a `key` here, given that we're not inside a loop?